packages/teleterm/src/services/tshd/createClient.ts

```diff
@@ -287,6 +287,21 @@ export default function createClient(
         });
       });
     },
+
+    async setGatewayLocalPort(gatewayUri = '', localPort = '') {
```

Comment:
I was cargo culting what was already there. 😶‍🌫️

One explanation that I came up with: this might make sense if we were trying to mimic Go API where an empty string is the default value you'd use for something of type `string` since you cannot pass nil where a string is expected.

But for me the tshd client level is something that should be grounded in the JS world. Here I'd rather just force those args to be present.

I'm going to remove this function call but leave them for others as I'm not sure if we're free to do that.

Reply:
OK, let's do this.